cranelift/filetests/filetests/isa/x64/fp_sp_pc.clif

```diff
@@ -0,0 +1,46 @@
+test compile precise-output
+target x86_64
+
+function %fp() -> i64 {
+block0:
+    v0 = get_frame_pointer.i64
+    return v0
+}
+
+;   pushq   %rbp
+;   movq    %rsp, %rbp
+; block0:
+;   movq    %rbp, %rax
+;   movq    %rbp, %rsp
+;   popq    %rbp
+;   ret
+
+function %sp() -> i64 {
+block0:
+    v0 = get_stack_pointer.i64
+    return v0
+}
+
+;   pushq   %rbp
+;   movq    %rsp, %rbp
+; block0:
+;   movq    %rsp, %rax
+;   movq    %rbp, %rsp
+;   popq    %rbp
+;   ret
+
+function %return_pointer() -> i64 {
+block0:
+    v0 = get_return_pointer.i64
+    return v0
+}
+
+;   pushq   %rbp
+;   movq    %rsp, %rbp
+; block0:
+;   movq    %rbp, %rdi
+;   movq    8(%rdi), %rax
```

Comment:
I suspect we could avoid the extra copy here by defining the return-address intrinsic to load directly from `8(%rbp)`, i.e. creating an `Amode` with `rbp` as a base register.